src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -47,7 +50,7 @@ function renderButton(action, display, style, showText, asItem) {
   } else {
     output = (
       <Button
-        className="control-button"
+        className={`control-button ${isPressed ? 'active' : ''}`}
         type={controlTypeToButtonType[type] || 'default'}
```

Comment:
I think it's enough to check if is active and add the class active although all the buttons will take this class if they are active and in css just add style for active class on pressable buttons.